app/models/miq_policy_content.rb

```diff
@@ -3,6 +3,53 @@ class MiqPolicyContent < ApplicationRecord
   belongs_to :miq_event_definition
   belongs_to :miq_action
 
+  def self.seed
+    fixture_file = File.join(FIXTURE_DIR, "miq_policy_contents.yml")
+    policy_contents = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
```

Comment:
Maybe I'm strange but I read policy_contents as MiqPolicyContent objects, not as fixtures/attributes used to create those objects.  Maybe content_attributes or content_fixtures?